avahi-common/domain.c

```diff
@@ -210,7 +210,8 @@ char *avahi_normalize_name(const char *s, char *ret_s, size_t size) {
         } else
             empty = 0;
 
-        avahi_escape_label(label, strlen(label), &r, &size);
+        if (!(avahi_escape_label(label, strlen(label), &r, &size)))
```

Comment:
Is another pair of parentheses necessary? I think ``if (!avahi_escape_label(...))`` should be enough.

Reply:
It's consistent with all the other places where `avahi_escape_label` is used like that
```
avahi-common/domain.c:                    if (!(avahi_escape_label(buf, strlen(buf), &type, &type_size)))
avahi-common/domain.c:                if (!(avahi_escape_label(buf, strlen(buf), &domain, &domain_size)))
avahi-core/dns.c:            if (!(avahi_escape_label((char*) AVAHI_DNS_PACKET_DATA(p) + idx, n, &ret_name, &l)))
```
I'm not sure why this coding style was chosen but I don't think it should be changed here.

Reply:
okay, looks terrible, but could be changed later.